plsync/sites.py

```diff
@@ -79,6 +80,7 @@
     makesite('atl05','67.106.215.192', '2610:18:111:c002::',   'Atlanta_GA', 'US', 33.636700, -84.428100, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
     makesite('atl06','70.42.177.64',   '2600:c0b:2002:5::',    'Atlanta_GA', 'US', 33.636700, -84.428100, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
     makesite('ath03','193.201.166.128', '2001:648:25e0::',     'Athens', 'GR', 37.936400, 23.944400, user_list, count=4, v6gw='2001:648:25e0::129', nodegroup='MeasurementLabCentos'),
+    makesite('bcn01','91.213.30.192',   '2001:67c:137c:5::',   'Barcelona', 'ES', 41.3947047, 2.0083384, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
```

Comment:
The lat/lon you have listed here are out to 7 decimal places. The others in the document are only to 6, and [Jordan has some note](https://docs.google.com/document/d/1pgrRyIA9F82RKC3CnSq30y-rRZI1QizzU6HmBtMBww4/edit#heading=h.c7y6z1b0n416) in the deployment document that states to not go past 4 decimal places. 6 seems to be okay, since all the other entries are at 6. Maybe we should standardize on 6 places?

Also, where did you get the coordinates? Google OneBox (for a "bcn airport coordinates" search) has them at 41.2974° N, 2.0833° E, and Wikipedia has them at 41.296944, 2.078333. Yours here seem a bit different.

Reply:
I got the coordinates by searching Google Maps for Barcelona. Will adjust the number of decimal places and change the coordinates to those that you provided.